internal/controller/management/projects/projects.go

```diff
@@ -134,6 +135,30 @@ type reconciler struct {
 		client.Object,
 		...client.CreateOption,
 	) error
+
+	createClusterRoleFn func(
+		context.Context,
+		client.Object,
+		...client.CreateOption,
+	) error
+
+	createClusterRoleBindingFn func(
+		context.Context,
+		client.Object,
+		...client.CreateOption,
+	) error
+
+	deleteClusterRoleFn func(
+		context.Context,
+		client.Object,
+		...client.DeleteOption,
+	) error
+
+	deleteClusterRoleBindingFn func(
+		context.Context,
+		client.Object,
+		...client.DeleteOption,
+	) error
 }
```

Comment:
I know we're moving away from this method of mocking out specific bits of functionality during tests, but this reconciler is stuck in the past and I wasn't about to refactor the whole thing.